addons/web/controllers/view.py

```diff
@@ -25,3 +34,83 @@ def edit_custom(self, custom_id, arch):
             ))
         custom_view.write({'arch': arch})
         return {'result': True}
+
+    @route('/web/view/save_multi', type='jsonrpc', auth="user")
+    def save_multi(self, model, ids, changes, context=None):
```

Comment:
Did we think about making this a server/ORM functionality rather than a client route?
Why would the feature only be useful from the client side, and not for server/automated actions etc also?

```python
self.env['account.budget.line'].search([]).write({
    'budget_amount': '* 2',
})
```